### PR TITLE
Add configs for building a bundle with Konflux

### PR DESCRIPTION
**Pull request by gbenhaim**
1. Add a script that Konflux will update with the latest references to the operands. This script is also used during the build for producing the variables that will be used when generating the manifests from templates.

2. Add a Containerfile for building the bundle.

### CODE REVIEW

File: operator/mtv-bundle.Containerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+FROM quay.io/konflux-ci/operator-sdk-builder@sha256:08f084228288d6314573d7742671fc7ddf84c19e4cf8eeb7623f261a514d0e4e as builder
```
Comment (mnecas) [inline operator/mtv-bundle.Containerfile:1]: Please move this under the /build/ dir

File: operator/mtv-bundle.Containerfile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+LABEL \
+        com.redhat.component="mtv-operator-bundle-container" \
+        version="2.7.0" \
+        name="migration-toolkit-virtualization/mtv-operator-bundle" \
+        License="Apache License 2.0" \
+        io.k8s.display-name="Migration Toolkit for Virtualization" \
+        io.openshift.tags="migration" \
+        io.k8s.description="Migration Toolkit for Virtualization - Operator Bundle" \
+        summary="Migration Toolkit for Virtualization - Operator Bundle" \
+        maintainer="Migration Toolkit for Virtualization Team <migtoolkit-virt@redhat.com>"
```
Comment (mnecas) [inline operator/mtv-bundle.Containerfile:63]: Please switch to 
```
        maintainer="Forklift by Konveyor Community <forklift-dev@googlegroups.com>"
```

Comment (gbenhaim) [reply operator/mtv-bundle.Containerfile]: done

File: operator/mtv-bundle.Containerfile (hunk 1 above)
Comment (gbenhaim) [reply operator/mtv-bundle.Containerfile]: done

**Comment by gbenhaim** [comment]
Once we will get this PR merged we will update https://github.com/kubev2v/forklift/pull/1076 with the correct path to the Containerfile.

**Review by mnecas — APPROVED** [review]

**Pull request merged**